src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/service/NetworkWatchListService.java

```diff
@@ -104,4 +162,20 @@ public void tick() {
     private long now() {
         return System.nanoTime();
     }
+
+    private void addTimeouts(Set<Endpoint> renew, long key) {
+        if (!renew.isEmpty()) {
+            timeouts.computeIfAbsent(key, mappingFunction -> new HashSet<>())
+                    .addAll(renew);
+        }
+    }
+
+    @Getter
+    @NoArgsConstructor
+    @AllArgsConstructor
+    @Builder(toBuilder = true)
+    private static class EndpointContext {
```

Comment:
(nip) As for me `WatchListEntry` is a more suitable name.

Reply:
Renamed.